app/code/Magento/Wishlist/Test/Unit/Controller/Index/UpdateTest.php

```diff
@@ -27,124 +30,124 @@
 
     /**
      * @inheritdoc
      */
     protected function setUp()
     {
-        $this->formKeyValidator = $this->createMock(Validator::class);
-        $this->wishlistProvider = $this->createMock(WishlistProviderInterface::class);
-        $this->quantityProcessor = $this->createMock(LocaleQuantityProcessor::class);
-        $this->context = $this->createMock(Context::class);
-        $this->resultRedirect = $this->createMock(Redirect::class);
-        $this->resultFactory = $this->createPartialMock(ResultFactory::class, ['create']);
+        $this->formKeyValidatorMock = $this->createMock(Validator::class);
+        $this->wishlistProviderMock = $this->createMock(WishlistProviderInterface::class);
+        $this->quantityProcessorMock = $this->createMock(LocaleQuantityProcessor::class);
+        $this->contextMock = $this->createMock(Context::class);
+        $this->resultRedirectMock = $this->createMock(Redirect::class);
+        $this->resultFactoryMock = $this->createPartialMock(ResultFactory::class, ['create']);
+        $this->messageManagerMock = $this->createMock(ManagerInterface::class);
+        $this->objectManagerMock = $this->createMock(ObjectManagerInterface::class);
         $this->requestMock = $this->getMockBuilder(RequestInterface::class)
             ->setMethods(['getPostValue'])
             ->getMockForAbstractClass();
-        $this->objectManagerMock = $this->createMock(ObjectManagerInterface::class);
 
-        $this->context->expects($this->once())
+        $this->resultFactoryMock->expects($this->any())
+            ->method('create')
+            ->willReturn($this->resultRedirectMock);
+        $this->contextMock->expects($this->once())
             ->method('getResultFactory')
-            ->willReturn($this->resultFactory);
-        $this->context->expects($this->once())
+            ->willReturn($this->resultFactoryMock);
+        $this->contextMock->expects($this->once())
             ->method('getObjectManager')
             ->willReturn($this->objectManagerMock);
-
-        $this->resultFactory->expects($this->any())
-                              ->method('create')
-                              ->willReturn($this->resultRedirect);
-        $this->context->expects($this->any())
+        $this->contextMock->expects($this->any())
             ->method('getRequest')
             ->willReturn($this->requestMock);
-
-        $this->messageManager = $this->createMock(ManagerInterface::class);
-        $this->context->expects($this->any())
+        $this->contextMock->expects($this->any())
             ->method('getMessageManager')
-            ->willReturn($this->messageManager);
+           ->willReturn($this->messageManagerMock);
 
-        $this->updateController = new Update(
-            $this->context,
-            $this->formKeyValidator,
-            $this->wishlistProvider,
-            $this->quantityProcessor
+        $this->updateController = (new ObjectManagerHelper($this))->getObject(
```

Comment:
Extract this to separate variable:
`$objectManager = new ObjectManagerHelper($this);`